### PR TITLE
fixed template support. added Cipher option for sshd

### PR DESCRIPTION
**Pull request by hmm01i**
the config_template option was not implemented.

also added support for Ciphers
Per man
```
Ciphers
             Specifies the ciphers allowed for protocol version 2.  Multiple ciphers must be comma-separated.  The supported ciphers are “3des-cbc”, “aes128-cbc”, “aes192-cbc”, “aes256-cbc”, “aes128-ctr”, “aes192-ctr”,
             “aes256-ctr”, “arcfour128”, “arcfour256”, “arcfour”, “blowfish-cbc”, “rijndael-cbc@lysator.liu.se”, and “cast128-cbc”.  The default is:

                aes128-ctr,aes192-ctr,aes256-ctr,arcfour256,arcfour128,
                aes128-cbc,3des-cbc,blowfish-cbc,cast128-cbc,aes192-cbc,
                aes256-cbc,arcfour,rijndael-cbc@lysator.liu.se
```

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -0,0 +1 @@
+.puppet-lint.rc
```
Comment (aptituz) [inline .gitignore]: Could you remove this change from your pull-request?

**Comment by aptituz** [comment]
Thanks for the pull request. I'd be happy to merge it, except for the added gitignore on puppet-lint.rc. If you update the pull request I'll merge it.

**Comment by hmm01i** [comment]
removed the .gitignore. thanks.

**Comment by aptituz** [comment]
Merci! Merged.

**Pull request merged**